### ISSUE TITLE
Ubuntu 19.10 & Kodi 18.5: Incompatibility?

### ISSUE THREAD

**Issue by jamesgithub2049** [NapoleonWils0n/kodi-playercorefactory#112]
I ve been unable to use this for the last few months.

I have followed the installation instructions and successfully installed it previously (prior to the latest releases of Kodi and Ubunutu). I ve tried different clean builds of Ubuntu and Ubuntu MATE and older version of Kodi, but I am just unable to get this to work anymore (even on different computers).

I ve replaced the installation error message for “fdk-aac-dev” with following::

`sudo apt install fdkaac`

and

`sudo apt install libfdk-aac-dev`

Therefore, could someone please confirm if they are able to use this amazing project with a clean build of Ubuntu 19.10 & Kodi 18.5? Or could someone please confirm what versions they find this remains compatible with?

Thank you.

**Comment by NapoleonWils0n** [comment]
hi mate

sorry your having a bit of bother

the fdkaac codec shouldnt matter as most of the time the script just copies the audio and video

i just checked on latest ubuntu and kodi

when i ran pilfer i got an error

i updated pilfer by opening the terminal and running

```
pip3 install --upgrade --user git+https://github.com/NapoleonWils0n/pilfer.git
```

after updating pilfer it worked fine

i checked playing and recording from within kodi and on the command line
and it worked fine

see if that works

**Comment by NapoleonWils0n** [comment]
hi mate

the version i tested on linux was 17.6 my bad

18.2 and 18.6 version of kodi on the mac bith just crash when i try to use any external player

im not sure if thats just the macs new stupid security causing the crash
or if kodi 18 just crashes trying to use an external player

have to turn the kodi log on and see if i can see what the issue is

**Comment by NapoleonWils0n** [comment]
hi mate

found out what the problem is and how to fix it

heres the fix

1 - move the cursor over a link in kodi so its highlighted

2 - press c on the keyboard

3 - select play using

4 - the play menu will appear

5 - select an option like save url, play or record from the menu and it will work

**Comment by NapoleonWils0n** [comment]
hi mate

heres a video with the fix for kodi 18.6 

https://www.youtube.com/watch?v=Q9WvY53D980

**Comment by jamesgithub2049** [comment]
Thank you.
